README.md

```diff
@@ -67,6 +67,12 @@ These examples focus on the Amazon SageMaker Python SDK which allows you to writ
 - [TensorFlow and TensorBoard](sagemaker-python-sdk/tensorflow_resnet_cifar10_with_tensorboard)
 - [Distributed TensorFlow](sagemaker-python-sdk/tensorflow_distributed_mnist)
 
+### Amazon SageMaker
```

Comment:
Can we make the header in Line 70 more descriptive?

Reply:
Whoops, yep, finished this line. "Using Amazon SageMaker with Apache Spark"